File: Modul 3/Bahasa C/PRAK304-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {

    int bilangan;

    printf("Input\n");
    scanf("%d", &bilangan);
    printf("\nOutput\n");
  if(bilangan < 100 && bilangan >=20) { 
    printf("Puluhan\n");
  }
  else if(bilangan <20 && bilangan >= 10) { 
    printf("Belasan\n");
  }
  else if(bilangan >0 && bilangan <= 9) { 
    printf("Satuan\n"); 
  }
  else if(bilangan == 0) { 
    printf("Nol\n");
  }
  else {
    printf("Anda Menginput Melebihi Limit Bilangan\n");
  }  
  return 0;
}

File: Modul 3/Bahasa Python/PRAK304-2310817120001-DinaIzzatiElfadheya.py
print("Input")
bilangan = (int(input()))

print("\nOutput")
if bilangan <100 and bilangan >=20:
  print("Puluhan")
elif bilangan <20 and bilangan >=10:  
  print("Belasan")
elif bilangan >0 and bilangan <=9: 
  print("Satuan")  
elif bilangan == 0 :
  print("Nol")
else:
  print("Anda Menginput Melebihi Limit Bilangan")
